### PR TITLE
Add ajax sample and reference source for script management

### PR DESCRIPTION
**Pull request by twsouthwick**
A subsequent change will attempt to hook them up. For now, things
compile and run. However, the post back still occurs in the sample
because we haven't actually hooked things up.

### CODE REVIEW

File: src/Extensions/Script/Serialization/JavaScriptSerializer.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// MIT License.
+
+using System.Text;
+
+namespace System.Web.Script.Serialization;
+
+// TODO: We'll want to use System.Text.Json and change it to be strongly typed
```
Comment (birojnayak) [inline src/Extensions/Script/Serialization/JavaScriptSerializer.cs:7]: please create an issue for us to remember.. as you the perf benefit.

File: src/Extensions/UI/ScriptComponentDescriptor.cs [hunk 2]
```diff
@@ -0,0 +1,386 @@
+//------------------------------------------------------------------------------
```
Comment (birojnayak) [inline src/Extensions/UI/ScriptComponentDescriptor.cs:1]: mark TODO to change

File: src/Extensions/UI/ScriptControlDescriptor.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
```
Comment (birojnayak) [inline src/Extensions/UI/ScriptControlDescriptor.cs:1]: same as above

File: src/Extensions/UI/ScriptDescriptor.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+//------------------------------------------------------------------------------
```
Comment (birojnayak) [inline src/Extensions/UI/ScriptDescriptor.cs:1]: same as above

**Review by birojnayak — COMMENTED** [review]
There is one concern w.r.t Linux other than minor comments. This won't work fully, I believe you are just getting this started.

File: src/Extensions/Script/Serialization/JavaScriptSerializer.cs (hunk 1 above)
Comment (twsouthwick) [reply src/Extensions/Script/Serialization/JavaScriptSerializer.cs:7]: I'm tracking them all in #9

File: src/WebForms/WebForms.csproj [hunk 5]
```diff
@@ -32,6 +32,7 @@
     <PackageReference Include="System.CodeDom" Version="8.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.SystemWebAdapters" Version="1.3.2" />
     <PackageReference Include="Microsoft.AspNetCore.SystemWebAdapters.CoreServices" Version="1.3.2" />
+    <PackageReference Include="System.Drawing.Common" Version="8.0.1" />
```
[the remark this answers is not in the document]
Comment (twsouthwick) [reply src/WebForms/WebForms.csproj:35]: no, but is currently required. see #125 for tracking removal it. I believe we're only using attributes and the Color enum, so we should be ok.

**Comment by twsouthwick** [comment]
> This won't work fully, I believe you are just getting this started.

Correct - it won't. THings will compile now, but won't work until #9 is done. And the list there will be ever growing for a bit as I chip away at it.....

**Review by birojnayak — APPROVED** [review]

**Pull request merged**